applications/asset_tracker_v2/src/events/modem_module_event.h

```diff
@@ -121,6 +121,7 @@ enum modem_module_event_type {
 	MODEM_EVT_CARRIER_REBOOT_REQUEST,
 	MODEM_EVT_CARRIER_EVENT_LTE_LINK_UP_REQUEST,
 	MODEM_EVT_CARRIER_EVENT_LTE_LINK_DOWN_REQUEST,
+	MODEM_EVT_CARRIER_EVENT_LTE_LINK_OFF_REQUEST
```

Comment:
Should this be `MODEM_EVT_CARRIER_EVENT_LTE_POWER_OFF_REQUEST` to be closer to the `LWM2M_CARRIER_EVENT_LTE_POWER_OFF` event and `lte_lc_power_off()` being called.

Reply:
Fair point, it should

Reply:
Renamed as described